api/api/migrations/0001_initial.py

```diff
@@ -49,18 +49,28 @@ class Migration(migrations.Migration):
             ],
         ),
         migrations.CreateModel(
-            name='Customer',
+            name='Pentester',
```

Comment:
why not create a new migration instead? bc of this i will have to delete prod database xD

Reply:
J'ai commit d'un coup donc si je checkout je vais perdre mes autres modifs. Comment je peux faire?